aten/src/ATen/native/MathBitsFallback.h

```diff
@@ -60,6 +63,18 @@ struct MathOpFallback {
             "If you got this error on a core op, please report a bug to PyTorch.");
         } else {
           is_write = alias_info->isWrite();
+          auto& ivalue = (*stack)[stack_start + i];
+          //TODO: add a table listing all possible cases and clearly state what works/is allowed
+          if (ivalue.isTensor()) {
+            auto mut_arg_tensor = ivalue.toTensor();
```

Comment:
this incurs a refcount bump. Use `const auto&`

Reply:
yup done!